### ISSUE TITLE
jedi-vim showing too many popups

### ISSUE THREAD

**Issue by strin** [davidhalter/jedi-vim#531]
<img width="277" alt="screen shot 2016-01-18 at 12 05 54 pm" src="https://cloud.githubusercontent.com/assets/2183232/12401538/e21214b2-bddb-11e5-8fb2-ee0300b2908d.png">

My .vimrc only has 

```
syntax on                           " syntax highlighing
filetype on                          " try to detect filetypes
filetype plugin indent on    " enable loading indent file for filetype
set rtp+=~/.vim/bundle/vundle/
call vundle#rc()

" jedi-vim: awesome Python autocompletion.
Plugin 'davidhalter/jedi-vim'

```

**Comment by blueyed** [comment]
Confirmed.

Seems to be a Jedi issue though.. `signatures` at https://github.com/davidhalter/jedi-vim/blob/625c568b1249db8c5f1b06a1f9371c06ba2d08b5/jedi_vim.py#L359 is `[<CallSignature: abspath index 0>, <CallSignature: abspath index 0>, <Ca
llSignature: abspath index 0>, <CallSignature: abspath index 0>, <CallSignature: abs
path index 0>]`.

**Comment by blueyed** [comment]
See https://github.com/davidhalter/jedi-vim/pull/532 for a quick attempt to fix this - although I'm not sure this should be handled in jedi-vim - possibly not.

**Comment by blueyed** [comment]
FWIW, getting only 3 (same) signatures currently.

**Comment by davidhalter** [comment]
Fixed since probably Jedi 0.15.
